packages/core/data-transfer/src/strapi/providers/local-destination/index.ts

```diff
@@ -229,23 +289,30 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
   async createAssetsWriteStream(): Promise<Writable> {
     assertValidStrapi(this.strapi, 'Not able to stream Assets');
 
+    if (!this.#areAssetsIncluded()) {
+      throw new ProviderTransferError('Attempting to transfer assets when they are not included');
+    }
+
     const removeAssetsBackup = this.#removeAssetsBackup.bind(this);
-    const strapi = this.strapi;
+    const thisStrapi = this.strapi;
```

Comment:
not sure why this change was implemented, I almost confused this with as a `this.strapi` typo